File: DivinoLanches/ClientApp/src/models/ComandaModel.ts
import { ProdutoComandaModel } from '../models/ProdutoComandaModel';

export class ComandaModel {

    id: Number
    nomeCliente: string
    produtos: ProdutoComandaModel[]
    valorTotal: string
    ativo: string
    formaPagamento: string

    constructor(obj: any) {
        this.id = obj.id
        this.nomeCliente = obj.nomeCliente
        this.produtos = obj.produtos
        this.valorTotal = obj.valorTotal
        this.ativo = obj.ativo
        this.formaPagamento = obj.formaPagamento
        return this
    }
}

File: DivinoLanches/Models/ComandaModel.cs
using System;
using System.Collections.Generic;

namespace DivinoLanches.Models
{
    public class ComandaModel
    {
        public int Id { get; set; }
        public string NomeCliente { get; set; }
        public List<ProdutoComandaModel> ListaProdutos { get; set; }
        public string ValorTotal { get; set; }
        public bool? Ativo { get; set; }
        public string FormaPagamento { get; set; }
    }
}
